sources/platform/actors/running/store.md

```diff
@@ -3,129 +3,125 @@ title: Actors in store
 
-![Free and rental Actors in Apify Store](./images/store/free_vs_paid_actors.png)
+![Rental Actor example](./images/store/rental-actor-example.png)
 
-Each rental Actor has a **free trial,** where the trial's length is always shown for each rental Actor.
+Each rental Actor has a _free trial_ period. The lenght of the trail is displayed in each Actor's page.
 
-![Rental Actors in Apify Store](./images/store/paid-actors-store.png)
+![Rental Actors in Apify Store](./images/store/rental-actor-trial-example.png)
 
----
+After a trial period, a flat monthly _Actor rental_ fee is automatically subtracted from your prepaid platform usage in advance for the following month. This fee goes directly to the developer and is paid on top of the platform usage generated by the Actor. You can read more about our motivation for releasing rental Actors in [this blog post](https://blog.apify.com/make-regular-passive-income-developing-web-automation-actors-b0392278d085/) from Apify's CEO Jan Čurn.
 
-![Rental Actors free trial](./images/store/paid-actors-trial.png)
 
-After a trial period, a flat monthly **Actor rental** fee is automatically charged from your prepaid platform usage in advance for the following month. This fee goes directly to the developer and is paid on top of the platform usage generated by the Actor. You can read more about our motivation for releasing rental Actors in [this blog post](https://blog.apify.com/make-regular-passive-income-developing-web-automation-actors-b0392278d085/) from Apify's CEO Jan Čurn.
+#### Rental Actors - Frequently Asked Questions
 
-**Can I run rental Actors via API or the Apify client?**
+##### Can I run rental Actors via API or the Apify client?
 
 Yes, when you are renting an Actor, you can run it using either our [API](/api/v2), [JavaScript](/api/client/js) or [Python](/api/client/python) clients as you would do with private or free public Actors.
 
-**Do I pay platform costs for running rental Actors?**
+##### Do I pay platform costs for running rental Actors?
 
-Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the monthly Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
+Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the per month Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
```

Comment:
```suggestion
Yes, you will pay normal [platform usage costs](https://apify.com/pricing/actors) on top of the monthly Actor rental fee. The platform costs work exactly the same way as for free public Actors or your private Actors. You should find estimates of the cost of usage in each individual rental Actor's README ([see an example](https://apify.com/drobnikj/crawler-google-places#how-much-will-it-cost)).
```

Reply:
Not sure why you changed that. It was fine (and better) as it was.

Reply:
it was to adhere to Msoft Style Guide, but after some consideration I think I will modify Vale setup to turn it off as it does not make much sense.